README.md

```diff
@@ -95,15 +95,15 @@ Step 1: Set environment variable in app service.
 After you deploy it, browse to the web app. You should see the secret on the web page.
 
 ## How to use AzureCliCredential
-There are 2 approaches to use AzureCliCredential. First way is create AzureCliCredential directly, the other way is use AzureCliCredential which is chained in DefaultAzureCredential.
-1. Create AzureCliCredential directly.
-```
+There are 2 approaches to use `AzureCliCredential`. First way is create `AzureCliCredential` directly, the other way is use `AzureCliCredential` which is chained in `DefaultAzureCredential`.
+1. Create `AzureCliCredential` directly.
+```C#
 using Azure.Identity;
 
 var credential = new AzureCliCredential();
 ```
-2. Use AzureCliCredential which is chained in DefaultAzureCredential.
-```
+2. Use `AzureCliCredential` which is chained in `DefaultAzureCredential`.
+```C#
```

Comment:
I usually use csharp, not sure if C# will actually work or not. Do you know?

Reply:
Yes, C# works, same as CSharp does